source/Nuke.Common/Tooling/ToolPathResolver.cs

```diff
@@ -120,7 +121,7 @@ private static string GetPackageDirectory(string[] packageIds, [CanBeNull] strin
                             .Concat(
                                 new[]
                                 {
-                                    EmbeddedPackagesDirectory == null
+                                    EmbeddedPackagesDirectory != null
```

Comment:
Pre-existing bug - it wasn't ever checking the `EmbeddedPackagesDirectory`

Reply:
fixed in 6.0.2